analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentServiceFactory.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.gm.analyzer.alignment;
+
+import java.util.Collections;
+
+public final class AlignmentServiceFactory {
```

Comment:
Why is the class `final`?

Reply:
https://stackoverflow.com/questions/32375149/is-it-mandatory-utility-class-should-be-final-and-private-constructor

Basically utility classes should be `final` (along with having a private constructor) in order to make it clear how they are supposed to be used. It's good practice, but I'll agree that it's actual usefulness is limited.

Reply:
OK, I never bothered but that makes sense. Learning new things every day! :)

Reply:
We all are :)